<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentWindow</class>
 <widget class="QDialog" name="PaymentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: white;</string>
  </property>
  <widget class="QLabel" name="total_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Total</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalNumber_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QComboBox" name="expMonth_box">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 2px solid black;</string>
   </property>
  </widget>
  <widget class="QLabel" name="cardInfo_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Enter Card Information</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cardNumber_edit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 2px solid black;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="expYear_box">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 2px solid black;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameOnCard_edit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 2px solid black;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="securityCode_edit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 2px solid black;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="zipcode_edit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 2px solid black;</string>
   </property>
  </widget>
  <widget class="QLabel" name="cardNumber_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Card Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="exp_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Expiration (MMYY)</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameOnCard_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Name On Card</string>
   </property>
  </widget>
  <widget class="QLabel" name="securityCode_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Security Code</string>
   </property>
  </widget>
  <widget class="QLabel" name="zipcode_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Zip Code</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
 border: 2px solid black;</string>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
